lib/recurly/apple-pay/apple-pay.js

```diff
@@ -127,30 +133,23 @@ export class ApplePay extends Emitter {
    * @private
    */
   configure (options) {
-    if (options.recurly) {
-      this.recurly = options.recurly;
-      delete options.recurly;
-    } else return this.initError = this.error('apple-pay-factory-only');
-
-    if (options.form) {
-      this.config.form = options.form;
-      delete options.form;
-    }
+    if (options.recurly) this.recurly = options.recurly;
+    else return this.initError = this.error('apple-pay-factory-only');
+
+    if (options.form) this.config.form = options.form;
+    if (options.recurring) this.config.recurring = options.recurring;
```

Comment:
nit: this being a boolean will prevent it from setting through consistently when the option is provided. The side-effect of this will result in success, since `false` and `undefined` are functionally equivalent to the way this option is interpreted.

Simple fix would be to do `if ('recurring' in options) /* ... */`